tests/utils.go

```diff
@@ -2037,7 +2038,12 @@ func NewRandomVMIWithNS(namespace string) *v1.VirtualMachineInstance {
 			Masquerade: &v1.InterfaceMasquerade{}}}}}
 
 	vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
-	vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("128Mi")
+	if isARM64() {
+		// UEFI boot need addition 128M for edk2
+		vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("256Mi")
```

Comment:
Can it be that we should move this to production code? The issue is really the uefi image which is huge for arm64 (I do not have enough knowledge in that area to know why it can be so small for x86_64 but but is so big for arm64).

I would therefore expect that we special-case that in the controller and request more memory on the pod if uefi boot is requested.

Reply:
Hi Roman,

It is a little bit complex. Qemu would receive 128M for UEFI boot and this memory is not counted in memory within VM, so I have added 
 128 Mi in getMemoryOverHead, this makes virt-launcher container have enough memory for qemu start VM via UEFI boot. I thought this should be enough.
https://github.com/kubevirt/kubevirt/blob/b1c258a3a27e047fb6b37f7a02054f8b91f38e32/pkg/virt-controller/services/template.go#L1857-L1862 

However, when doing test, I get following error when set `-m 128` in qemu command.
```
EFI stub: Booting Linux Kernel...
EFI stub: EFI_RNG_PROTOCOL unavailable, no randomness supplied
EFI stub: ERROR: Failed to relocate kernel
EFI stub: ERROR: Failed to relocate kernel
```
These error messages come after UEFI boot and kernel loaded. It seems that the kernel is doing firmware drivers check and get this error. I am still doing an investigation on this. I am not sure if it is appropriate to add this to the production code before I fully understand why it happens.

Reply:
> I would therefore expect that we special-case that in the controller and request more memory on the pod if uefi boot is requested.

Hi Roman, I spent some time investigating this issue. The efi driver is quite hard to understand. So I did tests for different images, like centos, ubuntu, and cirros. Not all images get the same error when boot in 128M memory with UEFI boot, like ubuntu shows "out of memory", but all images failed to boot in 128M memory.

To fix this we need to add additional memory to VM rather than the pods, like if user requests 128M for a VM, we need to increase memory for qemu-kvm up to 256M, and the user would see "256M" memory within the VM. I am not sure if it is appropriate.

Reply:
ok, thanks for the details. We can take the change like you did, but it may make sense to open an issue to track this.

Reply:
The issue has been created. https://github.com/kubevirt/kubevirt/issues/6363